praxis/contrib/gpu/scripts_gpu/te_helper.py

```diff
@@ -44,6 +47,10 @@ def get_fprop_caller_of_stack_transformer(fprop, deterministic):
     def set_layer_params_to_stack_transformer(stacked_transformer_obj, layer_p, layer_id):
         raise NotImplementedError
 
+    @staticmethod
+    def get_input_bld(original_bld, batch_axes, mdl_axis):
```

Comment:
What bld mean? Add a comment?

Reply:
I am not really know the means of `bld`, but this is used to specify the sharding pattern of inputs to TransformerLayers.
Refer to [transformer_models.py#L323]( https://github.com/google/praxis/blob/main/praxis/layers/transformer_models.py#L323).
Added a comment also.